src/dreyfus/src/clouseau_rpc.erl

```diff
@@ -285,6 +287,14 @@ analyze(Analyzer, Text) ->
 version() ->
     rpc({main, clouseau()}, version).
 
+-spec clouseau_major_vsn() -> null | binary() | no_return().
+clouseau_major_vsn() ->
+    case version() of
+        {ok, <<Major, _/binary>>} -> <<Major>>;
+        {ok, null} -> null;
+        {'EXIT', noconnection} -> throw("Clouseau is not connected yet!")
```

Comment:
The word "yet" sounds a bit odd.  It sounds like the reason for getting `noconnection` is because we have not yet configured the connection to Clouseau, while it might be happening because we have lost the connection to Clouseau that we had in the past.  Therefore it may make sense to lose the "yet" here.